pkg/network/protocols/grpc/monitor_test.go

```diff
@@ -80,10 +90,17 @@ func getClientsIndex(index, totalCount int) int {
 
 func (s *USMgRPCSuite) TestSimpleGRPCScenarios() {
 	t := s.T()
+
 	cfg := config.New()
 	cfg.EnableHTTP2Monitoring = true
+	cfg.EnableGoTLSSupport = s.withTls
+	cfg.BPFDebug = true
+
+	if s.withTls {
+		log.SetupLogger(seelog.Default, "debug")
+	}
```

Comment:
Should be reverted
please see `TestMain` in `tracer_test.go` how to make `DD_LOG_LEVEL` control the log level